### PR TITLE
add Victoria links

### PR DESCRIPTION
**Pull request by maniyax**
Update binaries.md.

### CODE REVIEW

File: doc/en/Binaries.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 * [Aleksandr](https://dl.bintray.com/olga-yakovleva/NVDA/RHVoice-voice-Russian-Aleksandr-4.2.6.nvda-addon), version 4.2.6
 * [Anna](https://dl.bintray.com/olga-yakovleva/NVDA/RHVoice-voice-Russian-Anna-4.1.6.nvda-addon), version 4.1.6
 * [Artemiy](https://rhvoice.su/downloads/?voice=artemiy&type=nvda), version 4.0.6
+* [Victoria](https://rhvoice.su/downloads/?voice=victoria&type=nvda), версия 4.0.7
 * [Arina](https://rhvoice.su/downloads/?voice=arina&type=nvda), версия 4.0.7
```
Comment (alex19EP) [inline doc/en/Binaries.md:18]:
```suggestion
* [Arina](https://rhvoice.su/downloads/?voice=arina&type=nvda), version 4.0.7
```

File: doc/en/Binaries.md [hunk 2]
```diff
@@ -14,6 +14,7 @@
...
 * [Pavel](https://rhvoice.su/downloads/?voice=pavel&type=nvda), версия 4.0.7
```
Comment (alex19EP) [inline doc/en/Binaries.md:19]:
```suggestion
* [Pavel](https://rhvoice.su/downloads/?voice=pavel&type=nvda), version 4.0.7
```

**Review by alex19EP — CHANGES_REQUESTED** [review]
it will be good if you replace all версия to version in the english document.

**Comment by maniyax** [comment]
Fixed.

**Review by alex19EP — APPROVED** [review]
LGTM thanks.

**Pull request merged**